compiler/cpp/src/thrift/generate/t_d_generator.cc

```diff
@@ -71,6 +71,14 @@ class t_d_generator : public t_oop_generator {
   }
 
 protected:
+
+  // D reserved words are suffixed with an underscore
+  static string suffix_if_reserverd(const string& name) {
```

Comment:
typo: `suffix_if_reserverd` should be `suffix_if_reserved`

Reply:
fixed, thanks